tests/docker_images/Dockerfile.testing

```
@@ -7,6 +7,7 @@ RUN pip install --upgrade pip
 
 ARG DEBIAN_FRONTEND=noninteractive
 RUN pip install requests
+RUN pip install imap_data_access
```

Are you still using `requests` within the container example, or just using imap_data_access which has all of its own dependencies already?

If you do need requests, then they should both be on the same line for a single Docker layer, not separate RUN commands, but if you don't need it, just remove that line.
```suggestion
RUN pip install imap_data_access
```

---

I need it for test_ialirt_processing_stack.py